packages/edit-site/src/components/dataviews/reset-filters.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * WordPress dependencies
+ */
+import { BaseControl, Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+export default ( { onChangeView } ) => {
+	return (
```

Comment:
This should also handle the text search, right?

Reply:
You're right, fixed at https://github.com/WordPress/gutenberg/pull/55955/commits/d78c2b252a7c55d0c9099d5f36c7a50419918216